src/runtime/parallel/README.md

```diff
@@ -1,11 +1,59 @@
 Parallelization is achieved by having multiple child processes running scenarios.
 
+#### Customizable work assignment
+Cucumber exposes customization of worker assignment via `setParallelCanAssign`.
+The example below overrides the default, `() => true` which processes test cases 
+indiscriminately, with a scheme that accepts untagged test cases as well as test cases
+where the first tag doesn't match the first tag of any in progress tests.
+
+```typescript
+import { setParallelCanAssign } from '@cucumber/cucumber'
+// Accept tests missing tags or no test is running having the same first tag
+setParallelCanAssign((pickleInQuestion, picklesInProgress) => _.isEmpty(pickleInQuestion.tags) 
+    || _.every(picklesInProgress, ({tags}) => _.isEmpty(tags) || tags[0].name !== picklesInProgress.tags[0].name))
+```
+* Example using the handler above
+* 2 workers, `A` and `B`
+* Scenarios tagged as `@simple` (2 secs) or `@complex` (3 secs)
+* The first tag of the scenarios: `[@complex, @complex, @complex, @simple, @simple, @simple]`
+
+| Time | WIP | Events |
+|---|---|---|
+| 0 |  | assigned `1 (@complex)` to `worker A` | 
+| 0 | `@complex` | skip `2 & 3 (@complex)` - assign `4 (@simple)` to `worker B` |
+| 2 | `@complex` | skip `2 & 3 (@complex)` - assign `5 (@simple)` to `worker B` |
+| 3 | `@simple` | assign `2 (@complex)` to `worker A` |
+| 4 | `@complex` | skip `3 (@complex)` - assign `6 (@simple)` to `worker B` |
+| 6 |  | assign `3 (@complex)` to `worker A` |
+| 9 |  | done |
```

Comment:
Thanks for the update! Thoughts on using this example as the feature test? For assertions I think we could parse the messages and validate the test case started / finished times to ensure no complex tests ran at the same time.

Reply:
Done! I totally coded to avoid parsing though. Flagged the workers A and B based on the first case they pick up. Then expected A to work complex in order (1-3) and B to work simple in order (1-3). There was a catch though with the third complex test. Both workers are ready when it's time to assign that test so there is no telling which one will pick it up, I took that into consideration.